src/modules/addLike.js

```diff
@@ -0,0 +1,29 @@
+import { getArt } from './homeDisplay.js';
+
+const likesURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/6MkPjjBjFG5Jhh37x3c8/likes';
+
+export const likeFn = async (id) => {
+  const data = { item_id: id };
+  try {
+    const response = await fetch(likesURL, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(data),
+    }); return await response.json();
+  } catch (error) {
+    return false;
+  }
+};
+
+export const getBtn = async () => {
+  await getArt();
+  const likeBtn = document.querySelectorAll('.like');
+  likeBtn.forEach((btn) => {
+    btn.addEventListener('click', (e) => {
+      const id = e.target.id.split('Like')[0];
+
+      likeFn(id);
+      setTimeout(() => window.location.reload(), 300);
+    });
+  });
+};
```

Comment:
- [ ] <b>REQUIRED:</b> You still haven't fixed the major issue, as I mentioned in the previous review, `Like` functionality is not functioning properly. when I click on like, instead of increasing likes as it requested by the project, it is making request and then refresh the page with no effect. Kindly fix it.